### ISSUE TITLE
update_dependencies in run script fails to detect dependencies on macOS

### ISSUE THREAD

**Issue by bartaz** [canonical-webteam/generator-canonical-webteam#76]
Originally reported as https://github.com/canonical-websites/snapcraft.io/issues/1057

When running ./run script on macOS dependencies are not correctly detected. Manually cleaning hash files or running ./run clean is required to install new dependencies.

```
./run serve
 
Yarn dependencies haven't changed. To force an update, delete .yarn.canonical-webteam-snapcraft-flask-048191a3.hash.
bash: md5: command not found
xargs: md5: No such file or directory
Pip dependencies haven't changed. To force an update, delete .pip.canonical-webteam-snapcraft-flask-048191a3.hash.
```

It seems that for some types of dependencies (pip at least: https://github.com/canonical-websites/snapcraft.io/blob/master/run#L322) `md5` command is run inside of docker image, while `md5_command` is detected in host system.

If host system uses different md5 command then ubuntu in docker, detecting dependencies will fail.

**Comment by tbille** [comment]
@bartaz have you seen this issue again, please reach out if you have.

**Comment by bartaz** [comment]
I don't think we had any dependency changes recently for me to notice, but I'm pretty sure it's still around.
